$OpenBSD: patch-make_common_NativeCompilation_gmk,v 1.2 2020/04/22 15:05:18 kurt Exp $

Keep full bin/lib with debug in debuginfo file so it can be restored
later in post-build. This results in jmod's without debuginfo so we
get the debug package without bloating the size of the jdk package.

Index: make/common/NativeCompilation.gmk
--- make/common/NativeCompilation.gmk.orig
+++ make/common/NativeCompilation.gmk
@@ -943,9 +943,7 @@ define SetupNativeCompilationBody
           # Setup the command line creating debuginfo files, to be run after linking.
           # It cannot be run separately since it updates the original target file
           $1_CREATE_DEBUGINFO_CMDS := \
-              $$($1_OBJCOPY) --only-keep-debug $$($1_TARGET) $$($1_DEBUGINFO_FILES) $$(NEWLINE) \
-              $(CD) $$($1_OUTPUT_DIR) && \
-                  $$($1_OBJCOPY) --add-gnu-debuglink=$$($1_DEBUGINFO_FILES) $$($1_TARGET)
+              $(CP) $$($1_TARGET) $$($1_DEBUGINFO_FILES)
 
         else ifeq ($(OPENJDK_TARGET_OS), macosx)
           $1_DEBUGINFO_FILES := \
